src/http-personas.md

```diff
@@ -1,4 +1,44 @@
 ---
 ---
 
-# Personas HTTP Interface
\ No newline at end of file
+# Personas HTTP Interface
+
+A persona, generated from xapi statements. 
+
+Can be accessed through the [connection API](/http-connection) and [restify api](/http-models).
+
+### Schema
```

Comment:
I think this has actually been changed. Might be worth discussing with @andrewhickey to see what he wants to do about documenting this.

Reply:
What was the verdict here @cbishopvelti?